packages/dicomImageLoader/codecs/jpeg.js

```diff
@@ -0,0 +1,1094 @@
+// jshint ignore: start
```

Comment:
PLEASE exclude this file - use the codecs from the codecs package, as this instance has no provenance, and is not reviewable.

Reply:
I'll do that later, there is no build for now. I need this to get merged so that the typescript gets added